zeth-contracts/contracts/BaseMixer.sol

```diff
@@ -57,9 +56,16 @@ contract BaseMixer is MerkleTreeMiMC7, ERC223ReceivingContract {
     uint constant jsOut = 2; // Nb of commitments/ciphertexts
 
     // We have 2 field elements for each digest (nullifierS (jsIn), commitmentS (jsOut), h_iS (jsIn) and h_sig)
-    // The root, v_pub_in and v_pub_out are all represented by one field element, so we have 1 + 1 + 1 extra public values
+    // The root, v_pub_in and v_pub_out are all represented by one field element, so we have 1 extra public values
```

Comment:
`v_pub_in` and `v_pub_out` are not represented by a field element anymore. Update comments accordingly to the changes you've made